subgraphs/yearn-v2/src/common/initializers.ts

```diff
@@ -1,330 +1,334 @@
-
-    vaultSnapshots.hourlyProtocolSideRevenueUSD = constants.BIGDECIMAL_ZERO;
-    vaultSnapshots.cumulativeProtocolSideRevenueUSD = constants.BIGDECIMAL_ZERO;
-
-    vaultSnapshots.hourlyTotalRevenueUSD = constants.BIGDECIMAL_ZERO;
-    vaultSnapshots.cumulativeTotalRevenueUSD = constants.BIGDECIMAL_ZERO;
-
-    vaultSnapshots.blockNumber = block.number;
-    vaultSnapshots.timestamp = block.timestamp;
-
-    vaultSnapshots.save();
-  }
-
-  return vaultSnapshots;
-}
-
-export function getOrCreateVault(
-  vaultAddress: Address,
-  block: ethereum.Block
-): VaultStore {
-  const vaultAddressString = vaultAddress.toHexString();
-  const vaultContract = VaultContract.bind(vaultAddress);
-
-  let vault = VaultStore.load(vaultAddressString);
-
-  if (!vault) {
-    vault = new VaultStore(vaultAddressString);
-
-    vault.name = utils.readValue<string>(vaultContract.try_name(), "");
-    vault.symbol = utils.readValue<string>(vaultContract.try_symbol(), "");
-    vault.protocol = constants.ETHEREUM_PROTOCOL_ID;
-    vault.depositLimit = utils.readValue<BigInt>(
-      vaultContract.try_depositLimit(),
-      constants.BIGINT_ZERO
-    );
-
-    const inputToken = getOrCreateToken(vaultContract.token());
-    vault.inputToken = inputToken.id;
-    vault.inputTokenBalance = constants.BIGINT_ZERO;
-
-    const outputToken = getOrCreateToken(vaultAddress);
-    vault.outputToken = outputToken.id;
-    vault.outputTokenSupply = constants.BIGINT_ZERO;
-
-    vault.outputTokenPriceUSD = constants.BIGDECIMAL_ZERO;
-    vault.pricePerShare = constants.BIGDECIMAL_ZERO;
-
-    vault.createdBlockNumber = block.number;
-    vault.createdTimestamp = block.timestamp;
-
-    vault.totalValueLockedUSD = constants.BIGDECIMAL_ZERO;
-    
-    vault.cumulativeSupplySideRevenueUSD = constants.BIGDECIMAL_ZERO;
-    vault.cumulativeProtocolSideRevenueUSD = constants.BIGDECIMAL_ZERO;
-    vault.cumulativeTotalRevenueUSD = constants.BIGDECIMAL_ZERO;
-    
-    vault.lastReport = constants.BIGINT_ZERO;
-    vault.totalAssets = constants.BIGINT_ZERO;
-
-    const managementFeeId =
-      utils.enumToPrefix(constants.VaultFeeType.MANAGEMENT_FEE) +
-      vaultAddress.toHexString();
-    let managementFee = utils.readValue<BigInt>(
-      vaultContract.try_managementFee(),
-      constants.DEFAULT_MANAGEMENT_FEE
-    );
-    utils.createFeeType(
-      managementFeeId,
-      constants.VaultFeeType.MANAGEMENT_FEE,
-      managementFee
-    );
-
-    const performanceFeeId =
-      utils.enumToPrefix(constants.VaultFeeType.PERFORMANCE_FEE) +
-      vaultAddress.toHexString();
-    let performanceFee = utils.readValue<BigInt>(
-      vaultContract.try_performanceFee(),
-      constants.DEFAULT_PERFORMANCE_FEE
-    );
-    utils.createFeeType(
-      performanceFeeId,
-      constants.VaultFeeType.PERFORMANCE_FEE,
-      performanceFee
-    );
-
-    vault.fees = [managementFeeId, performanceFeeId];
-
-    vault.save();
-  }
-
-  return vault;
-}
+import {
```

Comment:
you should probably revert changes to yearn

Reply:
Sure -  I will revert these.